app/transformers/api/current/patient_transformer.rb

```diff
@@ -4,12 +4,15 @@ def from_nested_request(payload_attributes)
       payload_attributes = payload_attributes.to_hash.with_indifferent_access
       address = payload_attributes[:address]
       phone_numbers = payload_attributes[:phone_numbers]
+      business_identifiers = payload_attributes[:business_identifiers]
       address_attributes = Api::Current::Transformer.from_request(address) if address.present?
       phone_numbers_attributes = phone_numbers.map { |phone_number| Api::Current::Transformer.from_request(phone_number) } if phone_numbers.present?
+      business_identifiers_attributes = business_identifiers.map { |business_identifier| Api::Current::Transformer.from_request(business_identifier) } if business_identifiers.present?
```

Comment:
Let's move the exceptions to the `current` namespace out into the inherited classes wherever the exceptions apply.

For example,

`business_identifiers` can be filtered away in the V2 transformer namespace.
`phone_numbers` can be filtered away in the V1 transformer namespace.
etc.

Kind of like `to_nested_response` in the Transformer inheritance hierarchy.

Reply:
Nevermind, figured that `business_identifiers` are optional even in V3.